src/main/kotlin/dev/fakek/FakeContext.kt

```diff
@@ -30,6 +30,7 @@ class FakeContext(private val faker: Faker = Faker.instance()) {
     private val fakerBook by lazy { faker.book() }
     private val fakerBoolean by lazy { faker.bool() }
     private val fakerColor by lazy { faker.color() }
+    private val fakerCompany by lazy { faker.company() }
```

Comment:
Thanks for keeping this in alphabetical order 😅  I will get a style guide put together one of these days.

Reply:
I saw that my last changes were reordered so I thought that would be a good idea to add it alphabetically